test_suite/cloud/test_azure.py

```diff
@@ -223,3 +223,31 @@ def test_services_running(self, host):
         with host.sudo():
             for service in service_list:
                 assert host.service(service).is_running
+
+    @pytest.mark.run_on(['rhel'])
+    def test_pkg_wanted(self, host):
```

Comment:
Not really important. But could you add a small docstring? (I get OCD when I see "None" in the description fields of the report :)